src/EmbeddedExplorer.ts

```diff
@@ -148,6 +150,16 @@ export class EmbeddedExplorer {
     return `${EMBEDDABLE_EXPLORER_URL}?${queryString}`;
   };
 
+  sendHandshakeToEmbed(message: string) {
+    this.embeddedExplorerIFrameElement.contentWindow?.postMessage(
+      {
+        name: HANDSHAKE_REQUEST,
+        message,
```

Comment:
lets make this message an object with a graphRef field. If there is no graphRef field, we know its a manual schema entry.